readthedocs/builds/forms.py

```diff
@@ -133,6 +138,7 @@ def __init__(self, *args, **kwargs):
             (None, '-' * 9),
             (BRANCH, BRANCH_TEXT),
             (TAG, TAG_TEXT),
+            (EXTERNAL, EXTERNAL_TEXT),
```

Comment:
Pretty sure we aren't meant to show this to users. I think we want https://github.com/readthedocs/readthedocs.org/blob/8e78d680d02aeba12644796b979ef62459f64932/readthedocs/builds/constants.py#L103